<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripleCutPlaneObjectSettingsWidget</class>
 <widget class="QWidget" name="TripleCutPlaneObjectSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="planeVisibilityGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Plane visibility (3D)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="sagittalPlaneCheckBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Sagittal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="coronalPlaneCheckBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Coronal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="transversePlaneCheckBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Transverse</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="viewAllPlanesButton">
          <property name="text">
           <string>All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="viewNoPlaneButton">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="interpolationGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Interpolation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="resliceInterpolationLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Reslice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="resliceInterpolationComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Nearest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Linear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cubic</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="displayInterpolationLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Display</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="displayInterpolationComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Nearest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Linear</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="sliceThicknessLabel">
          <property name="text">
           <string>Slice thickness:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sliceThicknessSpinBox">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>100</height>
           </size>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="volumeContributionGroupBox">
     <property name="title">
      <string>Image Mixer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
